src/app/home/home.component.css

```diff
@@ -0,0 +1,175 @@
+.box {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+   
+    
+  }
+  
+  .box div {
+      padding-left:15%;
+      padding-right:3%;
+      
+    width: 100%;
+    height: 100%;
+  }
+/* pogledati jos margine i ostaloo*/
+.search-bar{
+    margin-left: 16%;
+    padding-right: 5%;
+    margin-top:7%
+}
+
+
+.font-family{
+    font-family: 'Source Sans Pro', sans-serif;
+    font-weight: 600 !important;
+}
+/* for name ABH Restaurant */
+.font-size-20{
+    font-size: 20px;
+    font-weight: 600 !important;
+    
+}
+
+/* font 16px for nav-bar */
+.font-s-16{
+    padding: 0.5%;
+    font-size: 16px;
+ 
+}
+
+/* Header with backround image */
+header {
```

Comment:
Again, it's better to apply styles to css classes instead of html tags. Create a class for the header, for example `.homepage-header`